libs/langchain/langchain/schema/runnable/base.py

```diff
@@ -281,6 +282,10 @@ def invoke(self, input: Input, config: Optional[RunnableConfig] = None) -> Outpu
         """Transform a single input into an output. Override to implement."""
         ...
 
+    def __call__(self, input: Input, config: Optional[RunnableConfig] = None) -> Output:
```

Comment:
there are existing child classes that implement call with a different signature — seems relatively high risk, low reward given that? cc @nfcampos

Reply:
Yeah potentially problematic due to collisions with __call__